README.md

```diff
@@ -28,6 +28,11 @@ Use make to build a local copy of the register data
 
 Note, bikeshed works with Python 2.7, and not with Python 3.0.
 
+## Dockerised build
+
+    $ make docker-image
```

Comment:
Running this step I get an exception `Unable to find image 'registers-specification:latest' locally`. Any thoughts? It might be my Docker setup.